documentation/source/users/rmg/database/kinetics.rst

```diff
@@ -212,237 +69,7 @@ Each reaction family contains the files:
-    **Intra_2+2_cycloaddition_Cd**                        |f33|
-    **Intra_5_membered_conjugated_C=C_C=C_addition**      |f34|
-    **Intra_Diels_alder_monocyclic**                      |f35|
-    **Intra_Disproportionation**                          |f36|
-    **Intra_RH_Add_Endocyclic**                           |f37|
-    **Intra_RH_Add_Exocyclic**                            |f38|
-    **Intra_R_Add_Endocyclic**                            |f39|
-    **Intra_R_Add_ExoTetCyclic**                          |f40|
-    **Intra_R_Add_Exo_scission**                          |f41|
-    **Intra_R_Add_Exocyclic**                             |f42|
-    **Intra_Retro_Diels_alder_bicyclic**                  |f43|
-    **Intra_ene_reaction**                                |f44|
-    **Korcek_step1**                                      |f45|
-    **Korcek_step1_cat**                                  |f46|
-    **Korcek_step2**                                      |f47|
-    **Peroxyl_Disproportionation**                        |f48|
-    **Peroxyl_Termination**                               |f49|
-    **R_Addition_COm**                                    |f50|
-    **R_Addition_CSm**                                    |f51|
-    **R_Addition_MultipleBond**                           |f52|
-    **R_Recombination**                                   |f53|
-    **Singlet_Carbene_Intra_Disproportionation**          |f54|
-    **Singlet_Val6_to_triplet**                           |f55|
-    **SubstitutionS**                                     |f56|
-    **Substitution_O**                                    |f57|
-    **Surface_Abstraction**                               |f58|
-    **Surface_Adsorption_Bidentate**                      |f59|
-    **Surface_Adsorption_Dissociative**                   |f60|
-    **Surface_Adsorption_Double**                         |f61|
-    **Surface_Adsorption_Single**                         |f62|
-    **Surface_Adsorption_vdW**                            |f63|
-    **Surface_Bidentate_Dissociation**                    |f64|
-    **Surface_Dissociation**                              |f65|
-    **Surface_Dissociation_vdW**                          |f66|
-    **Surface_Recombination**                             |f67|
-    **intra_H_migration**                                 |f68|
-    **intra_NO2_ONO_conversion**                          |f69|
-    **intra_OH_migration**                                |f70|
-    **intra_substitutionCS_cyclization**                  |f71|
-    **intra_substitutionCS_isomerization**                |f72|
-    **intra_substitutionS_cyclization**                   |f73|
-    **intra_substitutionS_isomerization**                 |f74|
-    **lone_electron_pair_bond**                           |f76|
-    ===================================================== =====
-
+A full list of the kinetic families `can be found on the RMG website: <https://rmg.mit.edu/database/kinetics/families/>`_. .
```

Comment:
My only note here is that this page takes a long time to load (though it contains more info, so it is worth it). Maybe we try to speed up that page loading (cache it?), and/or put a warning here about long load times?

Reply:
Great! I think that is an appropriate solution for this PR.